var/spack/repos/builtin/packages/r-rgooglemaps/package.py

```diff
@@ -12,11 +12,14 @@ class RRgooglemaps(RPackage):
     background image to overlay plots within R. This requires proper coordinate
     scaling."""
 
-    homepage = "https://cran.r-project.org/package=RgoogleMaps"
-    url      = "https://cran.r-project.org/src/contrib/RgoogleMaps_1.2.0.7.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/RgoogleMaps"
+    homepage = "https://cloud.r-project.org/package=RgoogleMaps"
+    url      = "https://cloud.r-project.org/src/contrib/RgoogleMaps_1.2.0.7.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/RgoogleMaps"
 
+    version('1.4.3', sha256='44cb62bcd23e5b4807e91c5825352eb8d38aaaeb3b38a8271ca9f21c1e1d4b19')
+    version('1.4.2', sha256='b479996fcb72f067644a7ea7f00325e44e76efd202e84aaab022753c4a6d5584')
     version('1.2.0.7', '2e1df804f0331b4122d841105f0c7ea5')
 
+    depends_on('r@2.10:', type=('build', 'run'))
     depends_on('r-png', type=('build', 'run'))
-    depends_on('r-rjsonio', type=('build', 'run'))
+    depends_on('r-rjsonio@:1.2.0.7', type=('build', 'run'))
```

Comment:
I don't see this dependency listed

Reply:
I resolved this earlier. It is needed for version 1.2.0.7, which is the earliest version in spack. I looked a little further back in versions and found that it first shows up in version 1.2.0.5, but that predates the earliest spack package. I can specify it as 

`depends_on('r-rjsonio@1.2.0.5:1.2.0.7', type=('build', 'run'))`

but it needs to be in there for version 1.2.0.7.

Reply:
Oh wait, I have the spec wrong. It needs to be a `when` condition.

Reply:
@adamjstewart I have it correct now.

`depends_on('r-rjsonio', when='@1.2.0.5:1.2.0.7', type=('build', 'run'))`